dist/index.js

```diff
@@ -37,7 +37,7 @@ function update() {
   var colors = colorNamer(color.hex(), 'ntc')
 
   // Refresh list
-  colors.forEach(function(c) {
+  colors.ntc.forEach(function(c) {
```

Comment:
Any idea what caused this change?

Reply:
I have no idea. When I run the demo I got an error because the plugin return a object of arrays just like the README said. I had to add the `.ntc` index to acces it.
Should the plugin return only one array if there is only one name passed ?

Reply:
Hey sorry for the delay. I am not able to look into this just yet, but I will put it on my list to review some time this week. Stay tuned!